app/assets/stylesheets/application.scss

```diff
@@ -150,3 +150,12 @@ tr.negative {
   text-align: center;
   padding: 0.5rem 0 0.5rem;
 }
+
+.bank-question-search {
+  margin-bottom: 10px;
+  width: 30%;
+}
+
+.admin-question-buttons{
+  margin-top: 10px;
+}
```

Comment:
We've started using https://tailwindcss.com/. Could you utilize the utility CSS classes instead of generating new CSS classes?

Reply:
I switched over to using tailwind.